### ISSUE TITLE
How to calculate the Precision, Recall and F1-Measure in Tensorflow?

### ISSUE THREAD

**Issue by Gu-Youngfeng** [Gu-Youngfeng/LearningTensorflow#2]
The default loss functions don't include the **Precision, Recall** and **F1-Measure**, so it seems that we should find another method to implement these measures in Tensorflow.
